FREYA-1760: Data highlight and news on FunCoup6 (#440)
ScilifelabDataCentre/data.scilifelab.se: content/news/2025-10-16.md

---

> title: New data highlight on FunCoup 6, Large-scale functional association networks with regulatory links and integrated tools

Reviewer: ```suggestion
title: New data highlight on FunCoup 6: Large-scale functional association networks with regulatory links and integrated tools
```

date: 2025-10-16
thumbnail: "/img/news/FunCoup.png"

> caption: "Graphical Abstract of FunCoup 6 from [Buzzao et al.](https://doi.org/10.1093/nar/gkae1021)"

Reviewer: ```suggestion
caption: "Graphical abstract of FunCoup 6 from [Buzzao et al.](https://doi.org/10.1093/nar/gkae1021)"
```

show_thumbnail_on_page: True
images: ["/img/news/FunCoup.png"]
---

> The latest [data highlight](/highlights/) discusses a major update to the FunCoup platform: [FunCoup 6: advancing functional association networks across species with directed links and improved user experience.](/highlights/FunCoup6/) This iteration introduces regulatory links, enriched network coverage, and new analytical modules to deepen insight into gene/protein coupling across organisms.

Reviewer: ```suggestion
The latest [data highlight](/highlights/) discusses a major update to the FunCoup platform: [FunCoup 6: advancing functional association networks across species with directed links and improved user experience](/highlights/FunCoup6/). This iteration introduces regulatory links, enriched network coverage, and new analytical modules to deepen insights into gene and protein coupling across organisms.
```

> [FunCoup 6](https://funcoup.org/)’s enhancements include the application of a bin‑free Bayesian training framework over multiple species, enabling generation of directed links that reflect regulatory relationships, in contrast to prior undirected interaction networks. The update also integrates tools for pathway enrichment (via [ANUBIX](https://pubmed.ncbi.nlm.nih.gov/32788619/), [EASE](https://pubmed.ncbi.nlm.nih.gov/14519205/)) and disease or drug target module detection** using the [TOPAS](https://pubmed.ncbi.nlm.nih.gov/36458021/) algorithm. Comparative interactomics is elevated with a new mode of ortholog-based alignment queries. The website and API have been redesigned for improved accessibility.

Reviewer: ```suggestion
[FunCoup 6](https://funcoup.org/) introduces a bin‑free Bayesian training framework applied over multiple species, enabling generation of directed links that reflect regulatory relationships, in contrast to prior undirected interaction networks. The update also integrates tools for pathway enrichment (via [ANUBIX](https://pubmed.ncbi.nlm.nih.gov/32788619/) and [EASE](https://pubmed.ncbi.nlm.nih.gov/14519205/)), as well as disease and drug target module detection using the [TOPAS](https://pubmed.ncbi.nlm.nih.gov/36458021/) algorithm. Comparative interactomics are further enhanced with a new mode of ortholog-based alignment queries. The website and API have been redesigned for improved accessibility.
```

> This represents a substantial leap for the computational biology community: by combining regulatory links, enriched coverage, and modular analytics, FunCoup 6 moves toward more complete and multi-facetted views of functional networks. Researchers studying host–pathogen interactions, evolutionary network rewiring, or cross-species regulatory conservation will find enhanced utility in this resource.

Reviewer: ```suggestion
This represents a substantial leap for the computational biology community. By combining regulatory links, enriched coverage, and modular analytics, FunCoup 6 moves toward more complete and multifaceted views of functional networks. Researchers studying host–pathogen interactions, evolutionary network rewiring, or cross-species regulatory conservation will find enhanced utility in this resource.
```

To learn more, see the full data highlight: [FunCoup 6: advancing functional association networks across species with directed links and improved user experience.](/highlights/FunCoup6/)

>  If you have open network or computational biology work to showcase on the portal (e.g. network databases, cross‐species tools), please [get in touch with us](/contact/).

Reviewer: ```suggestion
 If you have open network or computational biology work to showcase on the portal (e.g. network databases or cross‐species tools), please [get in touch with us](/contact/).
```
